jocd/src/br/org/certi/jocd/core/Target.java

```diff
@@ -16,18 +16,28 @@
 package br.org.certi.jocd.core;
 
 import br.org.certi.jocd.dapaccess.DapAccessCmsisDap;
+import br.org.certi.jocd.dapaccess.Transfer;
 import br.org.certi.jocd.dapaccess.dapexceptions.Error;
-import br.org.certi.jocd.dapaccess.dapexceptions.TransferError;
 import br.org.certi.jocd.flash.Flash;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.TimeoutException;
 
 public class Target {
 
-  public static final int TARGET_RUNNING = 1;   // Core is executing code.
-  public static final int TARGET_HALTED = 2;    // Core is halted in debug mode.
-  public static final int TARGET_RESET = 3;     // Core is being held in reset.
-  public static final int TARGET_SLEEPING = 4;  // Core is sleeping due to a wfi or wfe instruction.
-  public static final int TARGET_LOCKUP = 5;    // Core is locked up.
+  public interface CoreRegisters {
```

Comment:
This interface identifier (and the others that implement it) should be named in singular form.
Using the plural form might cause misunderstanding (e.g.: a list of CoreRegister elements).